orcid-persistence/src/main/resources/db/updates/fix-null-relationship-on-funding-ext-ids.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+
+    =============================================================================
+
+    ORCID (R) Open Source
+    http://orcid.org
+
+    Copyright (c) 2012-2014 ORCID, Inc.
+    Licensed under an MIT-Style License (MIT)
+    http://orcid.org/open-source-license
+
+    This copyright and license information (including a link to the full license)
+    shall be included in its entirety in all copies or substantial portion of
+    the software.
+
+    =============================================================================
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+        
+    <changeSet id="fix_null_relationship_on_funding_ext_ids" author="Angel Montenegro" dbms="postgresql">
```

Comment:
Hi @wjrsimpson @rcpeters, 

Could you please take a look at this change set before merging it?

Thanks